Integrations/AttackIQFireDrill/AttackIQFireDrill.py

```diff
@@ -0,0 +1,475 @@
+        test_status = build_transformed_dict(raw_test_status, TEST_STATUS_TRANS)
+        test_status['Id'] = test_id
+        hr = tableToMarkdown(f'Test {test_id} status', test_status)
+        return_outputs(hr, {'AttackIQTest(val.Id === obj.Id)': test_status}, raw_test_status)
+    except HTTPError as e:
+        return_error(create_invalid_id_err_msg(str(e), ['500']))
+
+
+def build_test_results_hr(test_results, test_id):
+    """
+    Creates test results human readable
+    Args:
+        test_results (list): Results of the test (after being transformed)
+        test_id (str): ID of the test
+
+    Returns: Human readable of test results
+    """
+    keys = ['Assessment Name', 'Scenario Name', 'Hostname', 'Asset IP', 'Job State', 'Modified', 'Outcome']
+    test_results_mod = []
+    for t_res in test_results:
+        assessment = t_res.get('Assessment')
+        asset = t_res.get('Asset')
+        scenario = t_res.get('Scenario')
+        hr_items = {
+            keys[0]: assessment.get('Name'),
+            keys[1]: scenario.get('Name'),
+            keys[2]: asset.get('Hostname'),
+            keys[3]: asset.get('Ipv4Address'),
+            keys[4]: demisto.get(t_res, 'JobState.Name'),
+            keys[5]: t_res.get('Modified'),
+            keys[6]: demisto.get(t_res, 'Outcome.Name')
+        }
+        test_results_mod.append(hr_items)
+    return tableToMarkdown(f'Test Results for {test_id}', test_results_mod, keys)
+
+
+def get_test_results_command(args=demisto.args()):
+    """ Implements attackiq-get-test-results command
+    """
+    test_id = args.get('test_id')
+    params = {
+        'page': args.get('page', '1'),
+        'test_id': test_id,
+        'show_last_result': args.get('show_last_result') == 'True'
+    }
+    try:
+        raw_test_res = http_request('GET', '/v1/results', params=params)
+        test_res = build_transformed_dict(raw_test_res['results'], TEST_RESULT_TRANS)
+        hr = build_test_results_hr(test_res, test_id)
+        return_outputs(hr, {'AttackIQTestResult(val.Id === obj.Id)': test_res}, raw_test_res)
+    except HTTPError as e:
+        return_error(create_invalid_id_err_msg(str(e), ['500']))
+
+
+def get_assessments(page='1', assessment_id=None):
+    """
+    Fetches assessments from attackIQ
+    Args:
+        page (str or int): Page number to fetch
+        assessment_id (str): (Optional) If provided will fetch only the assessment with matching ID
+
+    Returns: Assessments from attackIQ
+    """
+    if assessment_id:
+        return http_request('GET', f'/v1/assessments/{assessment_id}')
+    return http_request('GET', '/v1/assessments', params={'page': page})
+
+
+def list_assessments_command():
+    """ Implements attackiq-list-assessments command
+    """
+    page = demisto.args().get('page_number')
+    try:
+        page_size = int(demisto.args().get('page_size', DEFAULT_PAGE_SIZE))
+        page = int(page)
```

Comment:
what if you get an exception on `page = int(page)`?
there code afterwards dose not handle it.

Reply:
Changed behaviour so it'll raise an error to the user in that case